src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -37,6 +38,18 @@ class SearchContainer extends React.Component {
       global.window.scrollTo(0, 0);
       this.loadSearch();
     }
+    this.setTitle();
+  }
+
+  setTitle() {
+    const {
+      location: { query },
+    } = this.props;
+    if (!query || isEmpty(query)) {
+      global.document.title = 'ResearchNow';
```

Comment:
I think that this should be "ResearchNow | NYPL"?

Reply:
That's not approved yet, I think